packages/cli/test/cli-codemod.spec.ts

```diff
@@ -44,4 +63,40 @@ describe('Stylable Cli Code Mods', () => {
         expect(stdout).to.match(/failed to parse/);
         expect(stdout).to.match(/CssSyntaxError: <css input>:1:20: Missed semicolon/);
     });
+
+    it('should report skipped files with no changes', () => {
+        populateDirectorySync(tempDir.path, {
+            'package.json': `{"name": "test", "version": "0.0.0"}`,
+            'style.st.css': `.root{}`,
+        });
+
+        const { stdout } = runCliCodeMod([
+            '--rootDir',
+            tempDir.path,
+            '--mods',
+            'st-import-to-at-import',
+        ]);
+
+        const dirContent = loadDirSync(tempDir.path);
+
+        expect(dirContent['style.st.css']).equal(`.root{}`);
+        expect(stdout).to.match(/Summery:\n\[CodeMod\] − .*?style\.st\.css/);
```

Comment:
not sure what the print is showing in this case

Reply:
it's this `−` that indicate skipped files